dhl_sdk/client.py

```diff
@@ -244,21 +252,29 @@ class DataHowLabClient:
 
     """
 
-    def __init__(self, auth_key: APIKeyAuthentication, base_url: str):
+    def __init__(
+        self, auth_key: APIKeyAuthentication, base_url: str, verify_ssl: bool = True
+    ):
         """
         Parameters
         ----------
         auth_key : APIKeyAuthentication
             An instance of the APIKeyAuthentication class containing the user's API key.
         base_url : str
             The URL address of the datahowlab application
+        verify_ssl : bool, optional
```

Comment:
If possible, I would consider following the same approach used by the `requests` package and allow either a boolean or a path to a certificate/custom CA ([reference](https://requests.readthedocs.io/en/latest/api/#requests.request)).

This way, users would still have the option to verify self-singed certificates or certificates issues by CAs that are not in their trust store.